### PR TITLE
Service stage export

### PR DESCRIPTION
**Pull request by ryanwoldatwork**

### CODE REVIEW

File: app/controllers/admin/service_stages_controller.rb [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
     def index
-      ensure_service_owner(service: @service, user: current_user)
-      @service_stages = @service.service_stages
+      if params[:service_id]
+        set_service
+        ensure_service_owner(service: @service, user: current_user)
+        @service_stages = @service.service_stages.order(:position)
+      else
+        ensure_service_manager_permissions
+        @service_stages = ServiceStage.all.order(:position)
```
Comment (ryanwoldatwork) [inline app/controllers/admin/service_stages_controller.rb:15]: export all Service Stages to csv

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -255,14 +255,14 @@ def form_permissions?(form:)
     return false if form.blank?
     return true if admin_permissions?
     return true if current_user.has_role?(:form_manager, form)
+    return true if form_approver_permissions?
```
Comment (ryanwoldatwork) [inline app/controllers/application_controller.rb:258]: this elevates the Organizational Form Approver to Form Manager status, as opposed to Response Viewer status.
this enables them to see all Form tabs

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -129,6 +129,7 @@ def question_type_javascript_params(question)
 
   def is_at_least_form_manager?(user:, form:)
     user.admin? ||
+      user.organizational_form_approver && user.organization_id == form.organization_id ||
```
Comment (ryanwoldatwork) [inline app/helpers/application_helper.rb:132]: says organizational_form_approver is also considered a form manager

File: app/models/service_stage.rb [hunk 4]
```diff
@@ -7,4 +7,19 @@ class ServiceStage < ApplicationRecord
...
   validates :name, presence: true
+
+  def self.to_csv
+    service_stages = ServiceStage.order('service_id', 'position')
+
+    example_attributes = ServiceStage.new.attributes
+    attributes = example_attributes.keys
+
+    CSV.generate(headers: true) do |csv|
+      csv << attributes + ['service name', 'service_provider_id', 'service_provider_name']
```
Comment (ryanwoldatwork) [inline app/models/service_stage.rb:18]: exports table columns + these lookup fields, for convenience

File: app/views/admin/service_stages/unscoped_index.html.erb [hunk 5]
```diff
@@ -0,0 +1,74 @@
+<% content_for :navigation_title do %>
```
Comment (ryanwoldatwork) [inline app/views/admin/service_stages/unscoped_index.html.erb:1]: `/service_stages/index.html.erb` assumes a Service be passed into it, 
such that the Service Stages are shown for a specific Service.

`unscoped...` is about seeing all Service Stages, regardless of Service

File: app/views/admin/services/_results.html.erb [hunk 6]
```diff
@@ -1,34 +1,30 @@
-<table class="usa-table">
+<table class="usa-table width-full">
```
Comment (ryanwoldatwork) [inline app/views/admin/services/_results.html.erb:1]: makes this table full width and re-orders some column, so name shows first

File: app/views/components/admin/_recent_responses.html.erb [hunk 7]
```diff
@@ -18,13 +18,13 @@
...
-            <%= link_to form.organization.name, admin_organization_path(form.organization) %>
+            <%= render "admin/organizations/badge", organization: form.organization %>
           </td>
           <td>
             <%= link_to form.name, admin_form_path(form) %>
           </td>
-          <td>
-            <%= form.submissions.where("created_at > ?", Time.now - days_since.days).count %>
+          <td data-sort-value="<%= c = form.submissions.where("created_at > ?", Time.now - days_since.days).count %>">
+            <%= number_with_delimiter(c) %>
```
Comment (ryanwoldatwork) [inline app/views/components/admin/_recent_responses.html.erb:27]: add comma to 1,000+ form count

**Review by MaroyaF — APPROVED** [review]

**Pull request merged**